qgis-app/styles/templates/styles/style_list.html

```diff
@@ -1,40 +1,57 @@
 {% extends 'styles/style_base.html' %}{% load i18n bootstrap_pagination humanize static thumbnail %}
 
+{% block extrajs %}
+{{ block.super }}
+<style>
+    table > thead > tr > th > a {
+        color: grey;
+    }
+    .sorted {
+        color: #006dcc;
+        font-size: 20px;
+    }
+    .sorted-name {
+        color: #006dcc;
+    }
+</style>
+{% endblock %}
+
+
 {% block content %}
     <h2>{% if title %}{{title}}{% else %}{% trans "All Styles" %}{% endif %}</h2>
 
     {% if not style_list %}
     <p>No data.</p>
     {% else %}
-        <p>{{ style_list.count }} record{{ style_list.count|pluralize:"s" }} found.</p>
+        <p>{{ count }} record{{ style_list.count|pluralize:"s" }} found.</p>
     <table class="table table-striped plugins">
         <thead>
             <tr>
                 <th></th>
                 <th>
                     Name
-                    <a href="{{ request.get_path }}?order_by=name"><i class="icon-arrow-up"></i></a>
-                    <a href="{{ request.get_path }}?order_by=-name"><i class="icon-arrow-down"></i></a>
+                    <a href="{{ request.get_path }}?order_by=name" {% if order_by == "name" %}class="sorted"{% endif %}><i class="icon-arrow-up"></i></a>
```

Comment:
@sumandari I think you can use template tag for this, https://docs.djangoproject.com/en/3.1/howto/custom-template-tags/. Then we can reuse for all order link.

Reply:
Thanks  a lot for your advice @dimasciput. I will learn it and replace my primitive approach 😅 on the template  with this one.